### PR TITLE
doc: CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by justinmk3**
- Mention `npm run test` with an example.
- Show screenshot of the artifact Download link.

## License

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by bryceitoc9 — APPROVED** [review]

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -83,11 +83,14 @@ Code from treating its files as project files.
...
-To run a single test in VSCode, run the `Extension Tests (current file)` launch-config.
-
-Alternatively:
+To run a single test in VSCode, do any one of:
 
+-   Run the _Extension Tests (current file)_ launch-config.
 -   Use Mocha's [it.only()](https://mochajs.org/#exclusive-tests) or `describe.only()`.
+-   Run in your terminal:
+    ```
+    NO_COVERAGE=true TEST_FILE=src/test/foo.test npm run test
```
Comment (awschristou) [inline CONTRIBUTING.md:92]: Is this a Mac/Linux only instruction, or does this work on Windows as well?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -139,7 +142,14 @@ To send us a pull request, please:
 GitHub provides additional document on [forking a repository](https://help.github.com/articles/fork-a-repo/) and
 [creating a pull request](https://help.github.com/articles/creating-a-pull-request/).
 
-### Importing icons from other open source repos
+## CI artifact
+
+Each commit and pull request is processed by an automated system which runs
+all tests and provides the build result via the _Details_ link as shown below.
+
+<img src="./docs/images/ci-artifact.png" alt="CI artifact" width="512"/>
```
Comment (awschristou) [inline CONTRIBUTING.md:150]: Why is this not using the markdown convention? (examples in extension-readme.md)

File: CONTRIBUTING.md (hunk 1 above)
Comment (bryceitoc9) [reply CONTRIBUTING.md:92]: Looks like this works in Powershell: `$Env:NO_COVERAGE = "true"; $Env:TEST_FILE = ".\src\test\cloudWatchLogs\commands\addLogEvents.test.ts"; npm run test`

The above works fine in MinGW bash.

File: CONTRIBUTING.md (hunk 2 above)
Comment (justinmk3) [reply CONTRIBUTING.md:150]: can't set width that way (except in some flavors of markdown, but not github markdown)

File: CONTRIBUTING.md (hunk 1 above)
Comment (justinmk3) [reply CONTRIBUTING.md:92]: Main point with this kind of doc is to give users the "secret" information. Not to list syntax variants (which is not secret).

File: CONTRIBUTING.md (hunk 2 above)
Comment (bryceitoc9) [reply CONTRIBUTING.md:150]: Is the width that important? Would it make more sense to just resize on your local machine?

File: CONTRIBUTING.md (hunk 1 above)
Comment (awschristou) [reply CONTRIBUTING.md:92]: We've had instructions in the repo in the past that caused confusion for people who tried to pull down and work with the repo. In a contributing doc (whose target audience is developers), why wouldn't we provide turn-key instructions that assist in the development process for all supported systems?

Comment (justinmk3) [reply CONTRIBUTING.md:92]: Depends on how verbose we are willing to make the docs.

File: CONTRIBUTING.md (hunk 2 above)
Comment (justinmk3) [reply CONTRIBUTING.md:150]: Width is important enough to use the `` tag, which is a vanishingly small cost.

I did resize the image itself, but the image editor disagrees with the web browser on what a width of 512 pixels looks like. In general also, having to resize the image physically is pretty awkward way to adjust layout in a readme file. Using `` is a reasonable practice.

File: CONTRIBUTING.md (hunk 1 above)
Comment (justinmk3) [reply CONTRIBUTING.md:92]: Can you check if this works in Windows (specifically with the `/` slashes and the lack of the `.ts` extension):

        $Env:NO_COVERAGE = "true"; $Env:TEST_FILE = "src/test/cloudWatchLogs/commands/addLogEvents.test"; npm run test

Comment (bryceitoc9) [reply CONTRIBUTING.md]: Confirmed working. Strangely, ran the same query with `.ts` appended and it didn't actually pick up files; the one you posted is more correct.

Comment (justinmk3) [reply CONTRIBUTING.md]: `TEST_FILE` wants the module name, i.e. without `.ts`. We should probably accept either in testRunner though.

**Pull request merged**